blog/2022-09-19-tauri-egui-0-1.mdx

```diff
@@ -0,0 +1,267 @@
+---
+authors: [lucasfernog]
+description: The Tauri team is happy to announce the tauri-egui 0.1.0 release.
+---
+
+# Announcing tauri-egui 0.1.0
+
+The Tauri team is pleased to announce the first release of [tauri-egui](https://github.com/tauri-apps/tauri-egui).
+
+[`egui`](https://github.com/emilk/egui) is a GUI library written in Rust. It leverages an OpenGL context via [glutin](https://github.com/rust-windowing/glutin).
+
+tauri-egui is a Tauri plugin that connects with the Tauri runtime event loop to allow you to create glutin windows via our [glutin fork](https://github.com/tauri-apps/glutin)
+and use egui through our [egui-tao](https://github.com/tauri-apps/egui) integration.
+
+## Setup
+
+The first step is to add the crate to your dependencies in Cargo.toml:
+
+```toml
+[dependencies]
+tauri-egui = "0.1"
+```
+
+Now you need to enable the plugin:
+
+```rust
+fn main() {
+  tauri::Builder::default()
+    .setup(|app| {
+      app.wry_plugin(tauri_egui::EguiPluginBuilder::new(app.handle()));
```

Comment:
this is not relevant to the blog post but why is it named `wry_plugin` when it got nothing to do with `wry`?

Reply:
it's a low level plugin (event loop) to the `tauri-runtime-wry` crate. `wry` exposes `tao`, so there's something to do with wry 😂 though i don't see any good name for this and too late to change it now.

Reply:
`event_loop_plugin` would be a good name but yeah too late now.